mm/bios_map.c

```diff
@@ -92,7 +92,8 @@ void bios_map_reserve(unsigned int from, unsigned int to)
 {
 	if(is_addr_in_bios_map(from)) {
 		bios_map_add(from, to, MULTIBOOT_MEMORY_AVAILABLE, MULTIBOOT_MEMORY_RESERVED);
-		reserve_pages(from, to);
+		if (page_table_size)
+			reserve_pages(from, to);
```

Comment:
Please, include curly braces in this condition.

As stated in the CODING file:
```
- Conditionals should always include curly braces, even if there is only a
  single statement within the conditional.
```

Reply:
Oops! Fixed.